transports/http/src/main/java/org/mule/transport/http/config/HttpsNamespaceHandler.java

```diff
@@ -23,24 +22,24 @@
 /**
  * Reigsters a Bean Definition Parser for handling <code><https:connector></code> elements.
  */
-public class HttpsNamespaceHandler extends AbstractMuleNamespaceHandler
+public class HttpsNamespaceHandler extends HttpNamespaceHandler
```

Comment:
The only thing HttpNamespaceHandler has different from AbstractMuleNamespaceHandler (beside the added methods) is the init method, and this one is overridden, so it´s ok. Take into account that this transport is being deprecated so it won´t evolve a lot in the future.